tracer/src/Datadog.Trace/HttpOverStreams/DatadogHttpClient.cs

```diff
@@ -226,9 +227,21 @@ async Task<bool> IsNewLine()
             }
             while (true);
 
+            var isChunked = false;
+            foreach (var encoding in headers.GetValues(TransferEncodingHeaderKey))
+            {
+                if ("chunked".Equals(encoding, StringComparison.OrdinalIgnoreCase))
```

Comment:
What's wrong with `string.Equals()`? 😝

Reply:
That's 8 more characters, are you trying to give me arthritis?! 😝